cpp/src/groupby/common/aggregation_requests.hpp

```diff
@@ -56,28 +56,27 @@ using SimpleAggRequestCounter = std::pair<AggRequestType, gdf_size_type>;
 
 static constexpr std::array<operators, 4> simple_aggregations = {SUM, MIN, MAX, COUNT};
 
-static constexpr std::array<operators, 2> complex_aggregations = {MEDIAN, QUANTILE};
+static constexpr std::array<operators, 2> ordered_aggregations = {MEDIAN, QUANTILE};
 
 
 /**---------------------------------------------------------------------------*
  * @brief  To verify that the input operator is part of simple_aggregations list.
  * Note that in this kind of aggregators can be computed in a single pass scan.
- * In the other hand compound aggregation like AVG need to be computed by simple
+ * In the other hand the compound aggregation MEAN need to be computed by simple
  * ones (SUM and COUNT).
  *---------------------------------------------------------------------------**/
 inline bool is_simple(operators op) {
-  return std::any_of(simple_aggregations.begin(), simple_aggregations.end(),
-                     [&](operators test) { return test == op; });
+  return std::find(simple_aggregations.begin(), simple_aggregations.end(), op) !=  std::end(simple_aggregations);
```

Comment:
Wasn't `std::find` only added in `C++17`?

Reply:
Here's a `constexpr` version:
```
template <class T, size_t N>
constexpr bool contains(std::array<T,N> const& haystack, T needle){
    for(auto i = 0u; i < N; ++i){
       if(haystack[i] == needle)
           return true;
    }
    return false;
}
```

Example: https://wandbox.org/permlink/tkAJVkAwTbG5I7pb

Reply:
> Wasn't `std::find` only added in `C++17`?

`std::find` has been around for yonks. cppreference says this version is "until C++20", and doesn't give a "since", so presumably it is old enough to not worry about it. `constexpr` versions, on the other hand, don't come until C++20.

Reply:
> std::find has been around for yonks.

I had thought so, but I was confused to see that https://en.cppreference.com/w/cpp/algorithm/find only has `since C++17` as the oldest standard mentioned in `std::find`.